xarray/core/concat.py

```diff
@@ -161,6 +161,45 @@ def concat(
     --------
     merge
     auto_combine
+
+    Examples
+    --------
+    >>> da = xr.DataArray(np.random.randn(2, 3),
```

Comment:
Unfortunately using random will cause the doctests to fail (unless we're careful about the seed, which probably isn't worthwhile vs using arange or similar)

Reply:
we do set a seed: https://github.com/pydata/xarray/blob/7152b41fa80a56db0ce88b241fbe4092473cfcf0/conftest.py#L37-L38
but I suspect the expected values were generated without it. It's still better to not rely on randomness at all, especially if the arrays are small enough to be created by hand:
```suggestion
    >>> da = xr.DataArray(
    ...     [[0, 1, 2], [3, 4, 5]],
```